src/Microsoft.Health.Dicom.Core/Features/CustomTag/CustomTagStoreEntry.cs

```diff
@@ -52,9 +52,18 @@ public CustomTagStoreEntry(int key, string path, string vr, string privateCreato
         /// </summary>
         public CustomTagStatus Status { get; set; }
 
+        /// <summary>
+        /// Convert to  <see cref="CustomTagEntry"/>.
+        /// </summary>
+        /// <returns>The custom tag entry.</returns>
+        public CustomTagEntry ToCustomTagEntry()
+        {
+            return new CustomTagEntry() { Path = Path, VR = VR, PrivateCreator = PrivateCreator, Level = Level, Status = Status };
+        }
+
         public override string ToString()
         {
-            return $"Key: {Key}, Path: {Path}, VR:{VR}, Level:{Level}, Status:{Status}";
+            return $"Key: {Key}, Path: {Path}, VR:{VR}, PrivateCreator:{PrivateCreator}, Level:{Level}, Status:{Status}";
```

Comment:
Where is this `ToString()` used? Is it simply for debugging?

Reply:
It's used here https://github.com/microsoft/dicom-server/blob/1addc3c0297b7defdde291a1dbbdd8986101038e/src/Microsoft.Health.Dicom.Api/Controllers/CustomTagController.cs#L47 for easy diagnosis from log.